sailfishusermanager.cpp

```diff
@@ -535,3 +536,53 @@ void SailfishUserManager::updateEnvironment(uint uid)
         file.close();
     }
 }
+
+QStringList SailfishUserManager::usersGroups(uint uid)
+{
+    m_timer->start();
```

Comment:
nit: I guess this is "wait another minute before exiting due to inactivity" timer restart - imo it could use a bit more descriptive timer name / abstraction / stte.

Reply:
Renamed